src/Collection/PermissionSets.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Friendica\Collection;
+
+use Friendica\BaseCollection;
+
+class PermissionSets extends BaseCollection
+{
+
```

Comment:
Isn't this one missing?

```suggestion
	/** @var PermissionSet[] */
	private $models;
```

Reply:
Since BaseCollection implements ArrayIterator, there isn't a `$models` property anymore.

Reply:
oups .. yes you're right ^^